harness/asyncGC.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2019  Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: >
+    Collection of functions used to capture references cleanup from garbage collectors
+---*/
+
+function asyncGc(target) {
```

Comment:
the file name is `asyncGC` but the function name is `asyncGc`. Let’s use `asyncGc` for both?

Reply:
thanks for the feedback, I agree with it. The only reason I'm not applying is that I'm currently suspending the work on this PR. See https://github.com/tc39/test262/pull/2329#issuecomment-528424274